error-handlers.js

```diff
@@ -0,0 +1,5 @@
+exports.errorHandler = (err, request, response, next) => {
```

Comment:
Consider giving this function a more specific name that indicates what kind of errors it is handling. As you will have several error handlers in your app (and you already have two!) this will make it clearer to anyone reading your code.